README.md

```diff
@@ -44,11 +44,13 @@ If you want to use GPUs be sure to follow the Kubernetes [instructions for enabl
 
 ## Quick Start
 
-Requirements
+### Requirements
 
-  * ksonnet version [0.8.0](https://github.com/ksonnet/ksonnet/releases) or later.
+  * ksonnet version [0.8.0](https://github.com/ksonnet/ksonnet) or later.
```

Comment:
I think https://ksonnet.io/#get-started is a better link.

Reply:
Done